distributed/dashboard/components/shared.py

```diff
@@ -486,6 +511,18 @@ def __init__(self, worker, height=150, **kwargs):
         )
         self.mem.line(source=self.source, x="time", y="memory")
         self.mem.yaxis.axis_label = "Bytes"
+        self.mem.add_layout(
+            LabelSet(
```

Comment:
I didn't really configure the label text at all; is there anything that could be done here to make it more visually consistent with the plots?